src/gt4py/next/program_processors/runners/dace_fieldview/gtir_builtin_translators.py

```diff
@@ -157,7 +168,30 @@ def _create_temporary_field(
     return Field(field_node, field_type)
 
 
-def translate_as_field_op(
+def extract_domain(node: gtir.Node) -> FieldopDomain:
+    """
+    Visits the domain of a field operator and returns a list of dimensions and
+    the corresponding lower and upper bounds.
```

Comment:
Since depending which range constructor in DaCe is used the upper bound is either inclusive (`tuple` based constructor) or exclusive (string based constructor) you should add which format the output follows.
I would also settle for one format (but I think this is already the case) and only deviate from it if necessary and make this clear.

Reply:
I have added the description of the returned bounds. Yes, the lowering follows the format with exclusive upper bound.